tests/test_abi.py

```diff
@@ -87,6 +87,21 @@ def test_selector(self):
         event = EventABI(name="FooEvent")
         assert event.selector == "FooEvent()"
 
+    def test_from_signature(self):
```

Comment:
Can this be parametrized with a few more examples?

Reply:
Done